modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -264,6 +264,67 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-174501. Verify that the only user with the right roles can see/change the state manager"
+	@priority = 5
+	test OnlyManagersCanSeeStateManager {
+		task ("Give I have a user with a manager role") {
```

Comment:
Hi @MGabriellaS, just a typo here. Could you please replace 'Give' with 'Given'? Thanks!